module.json

```diff
@@ -5,7 +5,7 @@
   "author": "Ourobor",
   "version": "0.5.5",
   "minimumCoreVersion": "0.6.0",
```

Comment:
This should be bumped to 0.8.6 so that it doesn't install for people still on 0.7.10 or earlier.